contracts/contracts/coordination/Coordinator.sol

```diff
@@ -39,6 +39,7 @@ contract Coordinator is Ownable {
         address provider;
         bool aggregated;
         bytes transcript;  // TODO: Consider event processing complexity vs storage cost
+        bytes publicKey;  // TODO: Consider event processing complexity vs storage cost
```

Comment:
I know this PR is still WIP, but looking forward to <https://github.com/nucypher/nucypher/issues/3081>, can we call this `blinding key` or something more specific? There will need to be another entry in here for the E2E protocol for something that equates to a `decryption request encrypting key` - see https://github.com/nucypher/nucypher/issues/3081#issuecomment-1523814876.